docs/install.md

```diff
@@ -84,45 +72,18 @@ kf marketplace
 
 ## Install a service broker
 Once you have the service catalog you'll want to install a service
-broker. You can use helm to install the gcp-service-broker from
-the third_party directory. 
-
-Configure GCP service account & APIs
-```
-gcloud iam service-accounts create \
-    gcp-service-broker
-
-gcloud iam service-accounts keys \
-    create /tmp/key.json --iam-account \
-    gcp-service-broker@$GCP_PROJECT.iam.gserviceaccount.com
-
-gcloud projects \
-    add-iam-policy-binding \
-    $GCP_PROJECT --member \
-    serviceAccount:gcp-service-broker@$GCP_PROJECT.iam.gserviceaccount.com \
-    --role "roles/owner"
-
-gcloud services enable \
-    cloudresourcemanager.googleapis.com \
-    iam.googleapis.com
-```
-
-Once you have your key.json, copy this in the values.yaml file
-in `/third_party/gcp-service-broker` file. 
-
+broker. This exampe will use a the `sc` cli tool and a hosted 
+set of service from GCP.
 
-Configure helm
+Install `sc` with homebrew
```

Comment:
Can we link to some docs that show how to install `sc`? For example, if I'm using linux then these docs aren't useful.